multiqc/modules/bustools/bustools.py

```diff
@@ -81,6 +81,12 @@ def prepare_data(self):
             "format": "{:,.0f}",
             "shared_key": "barcodes",
         }
+        self.headers["medianReadsPerBarcode"] = {
+            "title": "Median reads per barcode",
+            "scale": "RdYlGn",
+            "min": 0,
+            "format": "{:,.2f}",
```

Comment:
I think this should be the default so you don't really need it, but no big deal..